Working Mode.md

```diff
@@ -143,6 +143,14 @@ NOTE: A standard is a tool towards convergence, and changing a standard can ofte
 
 Implementation disagreement should not result in implementation-defined behavior or optional features.
 
+## Meetings
+
+The WHATWG doesn't generally run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liase with other groups.
```

Comment:
(typo)

```suggestion
The WHATWG doesn't generally run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liaise with other groups.
```

Reply:
Since this PR is already merged I've sent https://github.com/whatwg/sg/pull/178. Thanks @TimothyGu!